Contents/Code/update_tools.py

```diff
@@ -275,6 +278,73 @@ def simplify_title(self):
 
 
 class ArtistUpdateTool(UpdateTool):
+    def get_square_image(self, image_url):
+        """
+            Get square image from Audible
```

Comment:
Can the details from your PR message be added to the function comment so future contributors (and future me) know what is going on further into the function?

> crop each portrait image to a square centered at the top, and crop each landscape image at the horizontal center

Reply:
Just updated it!